scripts/firesim-setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Sets up FireSim for use as a library within REBAR
+
+set -e
+set -o pipefail
+
+RDIR=$(pwd)
+scripts_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+sims_dir=$scripts_dir/../sims/
+
+# Reenable the FireSim submodule
+git config --unset submodule.sims/firesim.update || true
+cd $sims_dir
+git submodule update --init firesim
+cd firesim
+./build-setup.sh $@ --library
```

Comment:
This seems to rebuild riscv-tools. Is this necessary, given that REBAR already packages all the tools?

Reply:
I figure that most people will use `./firesim-setup.sh --fast`, to get a prebuilt toolchain on AWS. I have yet to add a no-toolchain feature to build-setup.sh, but should do that in the future.